config.mk

```diff
@@ -12,10 +12,10 @@
 FIRM_ADDR_W ?=15
 
 #SRAM SIZE (LOG2)
-SRAM_ADDR_W ?=15
+SRAM_ADDR_W ?=7
```

Comment:
Why do you need to change this variable? The firmware uses more than 2**7 words of memory. If you want to try new values for debugging you should call the make command with those variables set, like so, 'make asic-synth SRAM_ADDR_W=7'. In any case, SRAM_ADDR_W set with 7 is not useful. Please revert this change.

Reply:
I reverted it to its original value
I changed it earlier but as it was pointed out in last PR I reverted all things in config.mk
this is the revert

Reply:
iob-soc/ master project has a default value for the SRAM_ADDR_W of 15.